### PR TITLE
Style tweaks for modal dialog

### PR DESCRIPTION
**Pull request by billz**

### CODE REVIEW

File: app/css/material-dark.php [hunk 1]
```diff
@@ -206,6 +206,7 @@ function lightenColor($color, $percent)
 
 .modal-body {
   background-color: <?php echo $backgroundColor; ?>;
+  min-height: 8rem;
```
Comment (marek-guran) [inline app/css/material-dark.php:209]: Light theme is missing this change

File: app/css/material-dark.php [hunk 2]
```diff
@@ -288,11 +289,11 @@ function lightenColor($color, $percent)
   border-bottom-right-radius: 18px!important;
   border-bottom-left-radius: 18px!important;
   position: relative;
-  margin-top: -18px;
```
Comment (marek-guran) [inline app/css/material-dark.php:291]: And also this chnage is missing in light theme

**Review by marek-guran — APPROVED** [review]
Looks OK to me, just light theme did not have 2 changes that are in dark theme.

File: app/css/material-dark.php (hunk 1 above)
Comment (billz) [reply app/css/material-dark.php:209]: Thanks. My changes brought back the card-body bottom radius issue. [36e8eec](https://github.com/RaspAP/raspap-webgui/pull/1454/commits/36e8eecbea73f819de716fb1ed26d144e0000705) should fix this.

**Pull request merged**